README.md

```diff
@@ -487,7 +487,7 @@ You can set the default rounding mode by passing one of the `BigDecimal` mode en
 ```ruby
```

Comment:
Do we want to update something more in the README? In the example code, for example, it passes `BigDecimal::ROUND_HALF_UP` to the `round` method, even though it's the new default.

Reply:
good catch! done! https://github.com/RubyMoney/money/pull/1144/commits/dc3f52bad482366acd2090652b54a1b729750b2e